apps/web/pages/api/l1-nfts/[address].ts

```diff
@@ -37,17 +32,12 @@ export default async function handler(
       address.toLowerCase(),
     );
 
-    const nfts = ownedNfts.map((nft) => {
-      return {
-        title: nft.title || nft.contract.name,
-        image:
-          nft.media && nft.media.length > 0
-            ? nft.media[0].thumbnail
-            : undefined,
-        tokenId: nft.tokenId,
-        contract: nft.contract.address,
-      };
-    });
+    let nfts: OwnedNft[] = [];
+    for await (const nft of alchemy.nft.getNftsForOwnerIterator(
+      address.toLowerCase(),
+    )) {
+      nfts.push(nft);
```

Comment:
What do you think @gershon ? IDK if it's a good thing that the API returns all the data or if it would be better to add a pagination system.